src/UI/App.hs

```diff
@@ -58,21 +55,26 @@ drawUI s = [vBox (renderWidget s (focusedViewName s) <$> focusedViewWidgets s)]
 renderWidget :: AppState -> ViewName -> Name -> Widget Name
 renderWidget s _ ListOfThreads = renderListOfThreads s
 renderWidget s ViewMail ListOfMails = vLimit (view (asConfig . confMailView . mvIndexRows) s) (renderListOfMails s)
-renderWidget s _ MailAttachmentOpenWithEditor = renderMailAttachmentOpenWithEditor s
-renderWidget s _ MailAttachmentPipeToEditor = renderMailAttachmentPipeToEditor s
+renderWidget s _ MailAttachmentOpenWithEditor =
+  renderEditorWithLabel (Proxy :: Proxy 'MailAttachmentOpenWithEditor) "Open with:" MailAttachmentOpenWithEditor s
```

Comment:
This is still a TODO to get rid of the second `Name` passed into the function. Thinking about using `HasName a` for this.